lib/pipely/deploy/client.rb

```diff
@@ -96,6 +97,8 @@ def create_pipeline(pipeline_name, definition, tags={})
           tags: default_tags.merge(tags)
         )
 
+        definition = yield(created_pipeline.id)
```

Comment:
For backwards-compatability, it would be nice if the method could still accept a definition string, or yield here if that is not supplied.